scripts/BUILD.bazel

```diff
@@ -12,6 +12,11 @@ bats_test(
     deps = [":gee-lib"],
 )
 
+shellcheck_test(
```

Comment:
shellcheck in the end is a linter, it can catch bugs before the code is even run.

I think we should have a `en_sh_library` or `en_sh_binary` target that as part of the build, runs linters.

Eg, as a pattern/policy, we should not learn about automatically detectable problems _after_ someone adds a test target, and actually runs it, we should learn about them at build time, as we're writing the code.

This is a very good change, though, approving!